### PR TITLE
I'll try again

### PR DESCRIPTION
**Pull request by redbrain**
fixing a glaring mistake in a previous (closed) PR

### CODE REVIEW

**Comment by redbrain** [comment]
wow im bad at this, expect me in a few minutes. maybe i  should locally host and test, but that's such a headache

File: src/commands/misc/httpdog.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+module.exports = class HttpDog extends Command {
+  constructor (client) {
+    super({
+      name: 'httpdog',
+      category: 'general',
+      parameters: [{
+        type: 'number',
+        required: false
```
Comment (Doges) [inline src/commands/misc/httpdog.js:10]: o

Comment (pedrofracassi) [reply src/commands/misc/httpdog.js:10]: It has a default value of `200`, hence it's not required. Also this PR has been closed already

Comment (Doges) [reply src/commands/misc/httpdog.js:10]: i just sent a fucking o

**Pull request closed without merging**